.buildkite/steps/test-acknowledgements-diff.sh

```diff
@@ -10,7 +10,7 @@ sed -e '$d' ACKNOWLEDGEMENTS.md > ACKNOWLEDGEMENTS-orig.md
 ./scripts/generate-acknowledgements.sh
 sed -e '$d' ACKNOWLEDGEMENTS.md > ACKNOWLEDGEMENTS-new.md
 
-if diff -q ACKNOWLEDGEMENTS-{orig,new}.md ; then
+if diff -u ACKNOWLEDGEMENTS-{orig,new}.md ; then
```

Comment:
Remove `-q` (quite) because we want to see the diff.
Add `-u` because we want it in unified format.